### DISCUSSION TITLE
Error: Missing environment variables (ACCESS_TOKEN_SALT and JWT_SECRET_KEY)

### DISCUSSION THREAD

**Discussion by Andreasair** [ghostfolio/ghostfolio#734] [Q&A]
[Nest] 1  - 03/02/2022, 4:01:25 PM     LOG [NestFactory] Starting Nest application...
================================
 Missing environment variables:
    ACCESS_TOKEN_SALT: undefined
    JWT_SECRET_KEY: undefined

 Exiting with error code 1
================================
[Nest] 1  - 03/02/2022, 4:01:37 PM     LOG [NestFactory] Starting Nest application...
================================
 Missing environment variables:
    ACCESS_TOKEN_SALT: undefined
    JWT_SECRET_KEY: undefined

 Exiting with error code 1
================================

**Comment by dtslvr** [comment]
Hi @Andreasair

You need an `.env` file in path defined in your `docker-compose.yml` file. You can find an example of the content here: [.env](https://github.com/ghostfolio/ghostfolio/blob/main/.env)

`ACCESS_TOKEN_SALT` and ` JWT_SECRET_KEY` needs to be random strings (secrets).

**Reply by Andreasair** [reply]
hi thanks for reply i am quite new. where to i place this env to my computer ? thanks

**Reply by dtslvr** [reply]
When you run this command, it assumes that the `.env` file is placed in the same folder (from where you are running this command):

```
docker run --env-file ./.env --port 3333:3333 ghostfolio/ghostfolio
```
